meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientInterface.java

```diff
@@ -79,6 +79,9 @@ enum ConnectState {
   class Stat {
     private final int _version;
     private final EntryMode _entryMode;
+    private long _expiryTime;
```

Comment:
Please consider add some comments here. They are in what time format? Are they relative time or absolute time etc

Reply:
Sure will do!